net-print/brother-mfc7820n-bin/metadata.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<!--
+# $Id$
```

Comment:
No longer needed.

Reply:
Is this (and the other such comments) referring to the "$Id$" lines?

Reply:
Yes, the _$Id$_ is an artifact from the time before the move to git. So the whole comment can be removed.